_includes/meta.html

```diff
@@ -22,19 +22,26 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <!-- Title and meta description
     ================================================== -->
-    {% if title %}
-    {% assign page_title = title | append : ' | ' %}
+
+    {% if seo_title %}
+      {% assign page_title = seo_title | append : ' | 18F ' %}
+    {% else %}
+      {% assign page_title = title | append : ' | 18F ' %}
+    {% endif %}
+
+    {% if description %}
+      {% assign page_description = description %}
```

Comment:
I may be missing something but It's not clear to me that we need this reassignment of the variable name. It seems like you could keep it as `description`, and remove the page part from "`page.description`.

Reply:
Heh, I didn't know you could do that, but I'm all for simplifying!